churaverse-plugins-client/src/synchroBreakPlugin/errors/synchroBreakPlayerCoinsNotFoundError.ts

```diff
@@ -0,0 +1,7 @@
+import { GamePluginError } from '@churaverse/game-plugin-client/errors/gamePluginError'
+
+export class SynchroBreakPlayerCoinsNotFoundError extends GamePluginError {
+  public constructor(playerId: string) {
+    super(`[シンクロブレイク] playerId: ${playerId} のコイン情報が存在しません`)
+  }
+}
```

Comment:
この辺りもSynchroBreakPluginErrorを継承して自動でprefixがつくようにできそう
他のエラークラスも同様
```suggestion
export class SynchroBreakPlayerCoinsNotFoundError extends SynchroBreakPluginError {
  public constructor(playerId: string) {
    super(`playerId: ${playerId} のコイン情報が存在しません`)
  }
}
```

Reply:
動作確認を行い、prefixをつけることができました！